doc/release-notes/release-6.sgml

```diff
@@ -111,7 +113,9 @@ This section gives an account of those changes in three categories:
 <sect1>Changes to existing options<label id="modifiedoptions">
 <p>
 <descrip>
-	<p>There have been no options changed.
+	<tag>--with-openssl</tag>
+	<p>OpenSSL features are now auto-enabled when OpenSSL 3.0.0
+	   or later is detected during build.
```

Comment:
AFAICT, branch commit 0f14f83d5e83e524291f4ba8ea2fac83e84722a3 removed the improvement summarized here. We should undo this SGML change AFAICT:

```suggestion
	<p>There have been no options changed.
```